server/services/claim-service.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
     const claimRecord = await claimRepository.create(claim)
 
     if (claim.mineType != null) {
-      for (let mineType in claim.mineType) {
+      let mineType
+      for (mineType in claim.mineType) {

```

Comment:
this could be rewritten as 
` for (const mineType in claim.mineType) {`

Reply:
Good tip! I hadn't realised that would be scoped within the for loop.

I've updated to use `const` and also changed `in` to `of` since `for .. in` is [not recommended for arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Why_Use_for...in).